### ISSUE TITLE
"Single source of truth" documentation

### ISSUE THREAD

**Issue by ghost** [google/closure-library#1064]
Is the documentation currently maintained/updated?

https://developers.google.com/closure/library/docs/overview
https://google.github.io/closure-library

Is there interest in having a single source of truth that people can point to? There are a few issues open regarding documentation: #968, #963, #856, #861

**Comment by shicks** [comment]
Neither is particularly relevant.  The developers.google.com documentation was written by someone outside our team years ago and we don't own it.  The github.io documentation was supposed to be a version that we owned, and we got a tech writer to work with us, but he bailed out before we really got it off the ground.  The idea was to scrub and mirror a bunch of our internal docs, but that has not actually happened.

At this point, it's not clear what makes the most sense w.r.t. documenting Closure, if ultimately the thing we want to recommend going forward is a new "Closure Good Parts" library that we extract out of it - it's going to look different enough that the docs should probably be started fresh for that, specifically.
